x-pack/plugins/apm/server/lib/apm_telemetry/collect_data_telemetry/tasks.ts

```diff
@@ -172,7 +172,7 @@ export const tasks: TelemetryTask[] = [
         AGENT_NAME,
         SERVICE_ENVIRONMENT,
         SERVICE_VERSION,
-        HOST_NAME,
+        HOST_HOSTNAME,
```

Comment:
do we need both?

Reply:
I don't know, probably yes. Maybe in a different issue?